pkg/deploy/che_configmap.go

```diff
@@ -198,6 +193,15 @@ func GetConfigMapData(cr *orgv1.CheCluster) (cheEnv map[string]string) {
 		CheMetricsEnabled:                      cheMetrics,
 	}
 
+	if cheMultiUser == "true" {
```

Comment:
You use the test :
```go
if cheMultiUser == "true" {
```
and in other cases the alternate:
```go
if cheMultiUser == "false" {
```
It seems to me it can lead to errors, since the `cheMultiuser` value is a string, and not a real `boolean`.
What will happen if the value added in the `customCheProperties` is neither `"false"` or `"true"` (think of case errors for example). Afaict this would lead to an inconsistent configuration.

I assume we should always use the test in the same way (compare with the same literal), or maybe the `GetCheMultiUser()` function should force the value to `"true"`, when it is not exactly `"false"` ?

Reply:
Actually `GetCheMultiUser()` returns default value which is `true`  when no value is set in CR

Reply:
But I do agree with you.

Reply:
> Actually GetCheMultiUser() returns default value which is true when no value is set in CR

Sure. I was more thinking of the case where an invalid value (not `false` nor `true`) would be set in the CR.

Reply:
Thanks for your fix in commit https://github.com/eclipse/che-operator/pull/185/commits/0e55f83db05c29c8a92e98c5e6993e343ef71bdd